onvm/go.sh

```diff
@@ -63,6 +65,13 @@ done
 
 verbosity_level="-v $verbosity"
 
+# If base virtual address has not been set by the user, set to default.
+if [[ -z $virt_addr ]]
+then
+    echo "Base virtual address set to default"
```

Comment:
[nit] might as well print what the default virt addr is, nobody really reads the usage. Looks good tho